metadata.json

```diff
@@ -64,6 +64,9 @@
       "operatingsystemrelease": [
         "18.04"
       ]
+    },
+    {
+      "operatingsystem": "FreeBSD"
```

Comment:
Please declare then non-EoL FreeBSD versions here

Reply:
that's ok